src/pyclass_init.rs

```diff
@@ -130,14 +130,14 @@ pub struct PyClassInitializer<T: PyClass> {
 }
 
 impl<T: PyClass> PyClassInitializer<T> {
-    /// Construct new initializer from value `T` and base class' initializer.
+    /// Constructs a new initializer from value `T` and base class' initializer.
     ///
-    /// We recommend to mainly use `add_subclass`, instead of directly call `new`.
+    /// We recommend to mainly use `add_subclass`, instead of directly calling `new`.
```

Comment:
While we're touching this docstring... in docs I tend to avoid "we" / "you" as much as I can and just have the abstract "It is recommended" and "the user should do xyz" instead.

```suggestion
    /// It is recommended to use `add_subclass` instead of this method for most usage.
```